model/src/main/antlr/com/here/ort/model/spdx/SpdxExpression.g4

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Shouldn't we add our license / copyright header also to this source file?

Reply:
Done.